src/locale/properties/shared/shared.en.properties

```diff
@@ -64,8 +64,10 @@ shared.partOf=Part of
 shared.onBehalfOf=via
 shared.versionOf=Version of
 shared.add=Add
+shared.export=Export
```

Comment:
seems like `shared.export` is not use on the code

Reply:
it is been used here https://github.com/ORCID/orcid-angular/blob/926d77fc187e278ad1ea5864b7c68a8510700c64/src/app/record/components/work-stack-group/work-stack-group.component.html#L65

Reply:
Yes, you are right is not been used anywhere. I am gonna remove it